cmd/appsctl/provision.go

```diff
@@ -54,9 +56,9 @@ var provisionBucketCmd = &cobra.Command{
 			return err
 		}
 
-		name := awsClient.AppsS3Bucket
+		name := awsclient.BucketWithDefaults("")
```

Comment:
s/`BucketWithDefaults`/`S3BucketWithDefaults` ? (to keep in line with the other `S3BucketExists`, `CreateS3Bucket`.